### PR TITLE
DDF-1602: Documentation updates for IdP server and client.

### PR DESCRIPTION
**Pull request by ricklarsen**
@stustison @pklinef

[Review on Reviewable](https://reviewable.io/reviews/codice/ddf/291)

### CODE REVIEW

**Comment by ricklarsen** [comment]
@clockard @vinamartin @djblue

**Comment by clockard** [comment]
:+1:

File: platform/security/docs/src/main/resources/ManContents.adoc [hunk 1]
```diff
@@ -109,3 +109,120 @@ The following packages have been exported by the DDF Security application and ar
 * Security PEP
 * Security PDP
 * Security STS
+
+=== IdP/SP
+
+{branding} The Security IdP application provides service provider handling that satisfies the http://docs.oasis-open.org/security/saml/v2.0/saml-profiles-2.0-os.pdf[SAML 2.0 Web SSO profile] in order to support external IdPs (Identity Providers).
```
Comment (vinamartin) [inline platform/security/docs/src/main/resources/ManContents.adoc]: This should be "The {branding} Security IdP....", right?

**Comment by vinamartin** [comment]
lgtm :+1:

**Comment by djblue** [comment]
:+1:

File: platform/security/docs/src/main/resources/ManContents.adoc [hunk 2]
```diff
@@ -109,3 +109,120 @@ The following packages have been exported by the DDF Security application and ar
...
+An internal Identity Provider solution.
+
+===== Configuration
+
+. Navigate to Admin Console -> DDF Security -> Configuration -> IdP Server
+. Click the + next to SP Metadata to add a new entry
+. Populate the new entry:
+.. with an HTTPS URL (https://),
+.. file URL (file:), or
+.. XML block to refer to desired metadata, e.g. (https://localhost:8993/services/idp/login/metadata)
```
Comment (stustison) [inline platform/security/docs/src/main/resources/ManContents.adoc]: This is the wrong URL, it should be https://localhost:8993/services/saml/sso/metadata

File: platform/security/docs/src/main/resources/ManContents.adoc [hunk 3]
```diff
@@ -109,3 +109,120 @@ The following packages have been exported by the DDF Security application and ar
...
+. Populate IdP Metadata field through one of the following:
+.. An HTTPS URL (https://)
+.. A file URL (file:)
+.. An XML block to refer to desired metadata (e.g., https://localhost:8993/services/idp/login/metadata)
+
+.IdP Client (SP) example.xml
+[source,xml,linenums]
+----
+<md:EntityDescriptor xmlns:md="urn:oasis:names:tc:SAML:2.0:metadata" entityID="https://localhost:8993/services/saml">
+  <md:SPSSODescriptor protocolSupportEnumeration="urn:oasis:names:tc:SAML:2.0:protocol">
```
Comment (stustison) [inline platform/security/docs/src/main/resources/ManContents.adoc]: It looks like we have the wrong metadata for each section. The IdP Client should be getting the IDPSSODescriptor metadata and the IdP Server should be getting the SPSSODescriptor metadata.

**Pull request merged**